docs/TPM_with_LPC_protocol.md

```diff
@@ -0,0 +1,820 @@
+        .SPIm_PWrite               (  1'b0                          ), // input
+        .SPIm_PWdata               ( 32'h0                          ), // input  [31:0]
+        .SPIm_Prdata               (                                ), // output [31:0]
+        .SPIm_PReady               (                                ), // output
+        .SPIm_PSlvErr              (                                ), // output
+
+        // Misc
+        .Device_ID                 ( Device_ID                      ), // input  [15:0]
+
+        // FBIO Signals
+        .FBIO_In                   (                                ), // output [13:0] <-- Do Not make any connections; Use Constraint manager in SpDE to sFBIO
+        .FBIO_In_En                (                                ), // input  [13:0] <-- Do Not make any connections; Use Constraint manager in SpDE to sFBIO
+        .FBIO_Out                  (                                ), // input  [13:0] <-- Do Not make any connections; Use Constraint manager in SpDE to sFBIO
+        .FBIO_Out_En               (                                ), // input  [13:0] <-- Do Not make any connections; Use Constraint manager in SpDE to sFBIO
+
+        // ???
+        .SFBIO                     (                                ), // inout  [13:0]
+        .Device_ID_6S              ( 1'b0                           ), // input
+        .Device_ID_4S              ( 1'b0                           ), // input
+        .SPIm_PWdata_26S           ( 1'b0                           ), // input
+        .SPIm_PWdata_24S           ( 1'b0                           ), // input
+        .SPIm_PWdata_14S           ( 1'b0                           ), // input
+        .SPIm_PWdata_11S           ( 1'b0                           ), // input
+        .SPIm_PWdata_0S            ( 1'b0                           ), // input
+        .SPIm_Paddr_8S             ( 1'b0                           ), // input
+        .SPIm_Paddr_6S             ( 1'b0                           ), // input
+        .FB_PKfbPush_1S            ( 1'b0                           ), // input
+        .FB_PKfbData_31S           ( 1'b0                           ), // input
+        .FB_PKfbData_21S           ( 1'b0                           ), // input
+        .FB_PKfbData_19S           ( 1'b0                           ), // input
+        .FB_PKfbData_9S            ( 1'b0                           ), // input
+        .FB_PKfbData_6S            ( 1'b0                           ), // input
+        .Sys_PKfb_ClkS             ( 1'b0                           ), // input
+        .FB_BusyS                  ( 1'b0                           ), // input
+        .WB_CLKS                   ( 1'b0                           )  // input
+
+    );
+```
+As one can see there is much more thant only clocks. There is one big disadvantage
+related to using "cell_macro" construct. This "cell-macro" is just a "black-box"
```

Comment:
@mgabryelski1 What you mean by `black-box`? Can't we find definition of cell_macro in symbiflow source? What you mean by `simulation is impossible`? Are you trying simulate FPGA built for EOS S3?